app/components/dashboard/DcChart.vue

```
@@ -5513,11 +5429,37 @@ const initChartCity = () => {
     })
     .elasticX(true);
 
-  mm.chartCity.xAxis().ticks(0);
-  if (mm.opt.chartCity.orderYmd) {
-    mm.chartCity.ordering(mm.orderYmd);
+    mm.chartCity.xAxis().ticks(0);
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(@stylistic/indent)